lib/Models/Catalog/CatalogItems/CartoMapV3CatalogItem.ts

```diff
@@ -0,0 +1,290 @@
+import { featureCollection, Geometry, GeometryCollection } from "@turf/helpers";
+import i18next from "i18next";
+import { computed, observable, runInAction } from "mobx";
+import RequestErrorEvent from "terriajs-cesium/Source/Core/RequestErrorEvent";
+import URI from "urijs";
+import JsonValue, {
+  isJsonNumber,
+  isJsonObject,
+  isJsonString,
+  isJsonStringArray,
+  JsonObject
+} from "../../../Core/Json";
+import loadJson from "../../../Core/loadJson";
+import Result from "../../../Core/Result";
+import TerriaError, { networkRequestError } from "../../../Core/TerriaError";
+import GeoJsonMixin, {
+  toFeatureCollection
+} from "../../../ModelMixins/GeojsonMixin";
+import CartoMapV3CatalogItemTraits from "../../../Traits/TraitsClasses/CartoMapV3CatalogItemTraits";
+import { GeoJsonTraits } from "../../../Traits/TraitsClasses/GeoJsonTraits";
+import TableStyleTraits from "../../../Traits/TraitsClasses/TableStyleTraits";
+import CreateModel from "../../Definition/CreateModel";
+import createStratumInstance from "../../Definition/createStratumInstance";
+import LoadableStratum from "../../Definition/LoadableStratum";
+import { BaseModel } from "../../Definition/Model";
+import StratumOrder from "../../Definition/StratumOrder";
+import Terria from "../../Terria";
+
+class CartoMapV3Stratum extends LoadableStratum(GeoJsonTraits) {
+  static stratumName = "cartoMapV3Stratum";
+  constructor(readonly catalogItem: CartoMapV3CatalogItem) {
+    super();
+  }
+
+  static load(item: CartoMapV3CatalogItem) {
+    return new CartoMapV3Stratum(item);
+  }
+
+  duplicateLoadableStratum(newModel: BaseModel): this {
+    return new CartoMapV3Stratum(newModel as CartoMapV3CatalogItem) as this;
+  }
+
+  // Hide "cartodb_id" style
+  @computed get styles() {
+    return [
+      createStratumInstance(TableStyleTraits, {
+        id: "cartodb_id",
+        hidden: true
+      })
+    ];
+  }
+}
+
+StratumOrder.addLoadStratum(CartoMapV3Stratum.stratumName);
+
+export default class CartoMapV3CatalogItem extends GeoJsonMixin(
+  CreateModel(CartoMapV3CatalogItemTraits)
+) {
+  static readonly type = "carto-v3";
+
+  @observable
+  geoJsonUrls: string[] = [];
+
+  @observable
+  mvtTileJsonUrls: string[] = [];
```

Comment:
Comment out until we add support? Otherwise implies that its supported already to a casual code reviewer...

Reply:
Done!